src/main/java/com/zaxxer/hikari/pool/PoolUtilities.java

```diff
@@ -56,7 +56,7 @@ public void quietlyCloseConnection(final Connection connection, final String clo
       final String addendum = closureReason != null ? " (" + closureReason + ")" : "";
       try {
          LOGGER.debug("Closing connection {} in pool {}{}", connection, poolName, addendum);
-         if (connection != null && !connection.isClosed()) {
+         if (!connection.isClosed()) {
```

Comment:
It is possible for *connection* to be null here, when called from the catch of *HikariPool.addConnection()*.